pkg/build/rpm_test.go

```diff
@@ -117,9 +121,110 @@ func TestCopyRPMsNoRPMDir(t *testing.T) {
 	require.NoError(t, err)
 	defer os.Remove(builder.eibBuildDir)
 
+	rpmSourceDir := filepath.Join(builder.buildConfig.ImageConfigDir, "rpms")
```

Comment:
Consider adding a constant to rpm.go with the name of the directory and then referencing that both here and within the RPM handling code itself. I could see us still needing to reorganize the image config directory at some point and it'll be simpler if we have all of the knowledge of how to deal with specific components contained in their respective source code.

Reply:
Another alternative, which I actually like better and would like to change in the custom scripts handling, would be to write a `generateRPMPath()` method in rpm.go that returns the full path, so that the knowledge of where it lives doesn't leak out of rpm.go. Then both rpm.go and rpm_test.go can use that method instead of having multiple places where we join the file paths.